### DISCUSSION TITLE
Option to exclude folder/files from site tree navigation

### DISCUSSION THREAD

**Discussion by e4rthdog** [requarks/wiki#6616] [Help / Questions] [answered]
I am using the Custom Navigation in order to present a fixed menu to my users and allow for all the content to be viewable by the Site Tree navigation.

The thing is that i don't want the content (folder and files) that i use for the static navigation, to appear in the Site Tree. Is there a way to Achieve this?

**Comment by NGPixel** [comment, marked as answer]
As answered on Discord, no.
